mon-pix/translations/fr.json

```
@@ -169,26 +169,21 @@
       "title": "Présentation",
       "assessment": {
         "title-with-username": "<span>{userFirstName}</span>,<br> commencez votre parcours",
-        "title": "Commencez votre parcours",
+        "title": "Commencez votre parcours Pix",
         "action": "Je commence",
-        "announcement": "Démarrez votre parcours d'évaluation personnalisé.'<br>' Inscrivez-vous ou connectez-vous sur la plateforme Pix et lancez votre test.",
+        "announcement": "Inscrivez-vous ou connectez-vous sur la plateforme Pix et lancez votre test.",
         "certify": {
-          "title": "Préparer une certification reconnue par l’État et le monde professionnel",
-          "description": "La certification Pix permet de valoriser ses compétences numériques auprès des employeurs."
+          "title": "Les réponses validées dans ce parcours contribueront à alimenter votre profil personnel de compétences numériques sur Pix.",
+          "description": " En fonction de votre situation et dans certaines conditions, vous pouvez accéder à la certification Pix, reconnue par le monde professionnel, qui permet de valoriser vos compétences numériques. Pour plus d’information, renseignez-vous auprès de l’organisateur du parcours."
```

```suggestion
          "description": " En fonction de votre situation et dans certaines conditions, vous pouvez accéder à la certification Pix, reconnue par le monde professionnel, qui permet de valoriser vos compétences numériques. Pour plus d’informations, renseignez-vous auprès de l’organisateur du parcours."
```

---

J'ai l'impression qu'on dit plus ça : 
```suggestion
          "description": " En fonction de votre situation et dans certaines conditions, vous pouvez accéder à la certification Pix, reconnue par le monde professionnel, qui permet de valoriser vos compétences numériques. Pour plus d’information, rapprochez-vous auprès de l’organisateur du parcours."
```

---

"rapprochez-vous de l'organisateur" non ? Le texte initial me parait clair.

---

Par contre c'est le scope de prescription non ?

---

Oui je propose "rapprochez-vous" plutôt que "renseignez-vous"

---

"rapprochez-vous auprès de" je trouve ça bizarre comme tournure 🤔

---

Ah yes carrément !